kconfig/Kconfig

```diff
@@ -81,7 +81,7 @@ menu "Common"
 
             choice
                 prompt "MAP-T"
-                default OSN_BACKEND_MAPT_NULL
+                default OSN_BACKEND_MAPT_CERNET
```

Comment:
This implementation is used. It represents the target layer of MAPTM. Keeping the flag OSN_BACKEND_MAPT_NULL instead of OSN_BACKEND_MAPT_CERNET will force the manager to use the empty APIs instead of the correct ones.
In our tests we used the implementation with the flag OSN_BACKEND_MAPT_CERNET.
Should we keep the NULL one in the PR?

Reply:
From open-source perspective:
* Default for MANAGER_MAPTM is 'n' therefore (in the default configuration) this implementation is not used anywhere.
* Hence, it (the default) should be set to OSN_BACKEND_MAPT_NULL.

In the kconfig configuration for your product (stored in the vendor layer), you set both the MANAGER_MAPTM and OSN_BACKEND to whatever you want.